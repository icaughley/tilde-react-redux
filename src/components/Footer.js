import React from 'react';
import "../style/style.css";

export default function Footer() {
    return (
        <div className="footer">
            Copyright &copy; 2017, Stock Software
        </div>
    );
}