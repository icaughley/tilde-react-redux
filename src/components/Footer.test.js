import React from "react";
import renderer from "react-test-renderer";
import {shallow} from "enzyme";
import Footer from "./Footer";

describe('Footer component', () => {
    it('has not changed', () => {
        expect(renderer.create(<Footer />)).toMatchSnapshot();
    });

    it('has the correct class', () => {
        const component = shallow(<Footer />);
        expect(component.hasClass('page-footer')).toEqual(true);
    });
});
