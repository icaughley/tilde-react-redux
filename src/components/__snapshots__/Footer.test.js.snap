exports[`Footer component has not changed 1`] = `
<div
  className="page-footer">
  Copyright Â© 2017, Stock Software
</div>
`;
